lib/tasks/migrations/migrations.rake

```diff
@@ -202,9 +202,13 @@ namespace :migrations do
       PreuploadPresentation: setting.get_value('Preupload Presentation'),
     }.compact
 
+
+    # Sets Record to default_enabled in V3 if set to optional in V2
+    rooms_config_record_value = infer_room_config_value(setting.get_value('Room Configuration Recording'))
```

Comment:
Wont `infer_room_config_value` return `default_enabled`?

Reply:
Nope, `default_enabled` has been completely removed from `inter_room_config_value`'s return.

**V2                          V3**
enabled or true -> true
disabled            -> false
optional             -> optional (default_disabled)

A clause has been added line 211: if record is "optional", it is re-written to "default_enabled"